app/stylesheets/main.scss

```diff
@@ -22,9 +30,11 @@ body, html {
   display: flex;
   width: 100%;
   height: 100%;
+  min-height: 100vh;
+  max-height: 100vh;
   flex-direction: column;
   font-size: var(--sn-stylekit-font-size-editor);
-
+  line-height: 1.5em;
```

Comment:
This PR should only address syntactical changes, and not functionality or appearance. This should be left to a separate PR.

Reply:
This is to fix issue https://github.com/sn-extensions/markdown-basic/issues/3. It's an easy fix. Are you sure you want to remove it?

Reply:
Yup. PRs should have only one area of focus. You can't make an architecture PR but also squeeze in a somewhat opinionated CSS change.

Reply:
OK

Reply:
Fixed https://github.com/sn-extensions/markdown-basic/pull/29/commits/966740e7811156ce580f59ad6474b0eef9b3239c